build_manylinux_wheels.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Builds manylinux cyipopt wheels with Ipopt 3.14.11 based on MUMPS 5.5.1, and OpenBLAS 0.3.15
+set -eu  # Stop script if a line fails
+TAG=${1}
+echo "Building cyipopt with tag $TAG"
+
+# Install efficient BLAS & LAPACK library
+yum install -y openblas-devel-0.3.15-4.el8
+
+pushd /tmp
+
+# MUMPS (Linear solver used by Ipopt)
+git clone https://github.com/coin-or-tools/ThirdParty-Mumps --depth=1 --branch releases/3.0.4
+pushd ThirdParty-Mumps
+sh get.Mumps
+./configure --with-lapack="-L/usr/include/openblas -lopenblas"
+make
+make install
+popd
+
+
+# Ipopt (The solver itself)
+git clone https://github.com/coin-or/Ipopt --depth=1 --branch releases/3.14.11
+pushd Ipopt
+./configure --with-lapack="-L/usr/include/openblas -lopenblas"
+make
+make install
+popd
+
+# build cyipopt for many python versions
+git clone https://github.com/mechmotum/cyipopt --depth=1 --branch $TAG
```

Comment:
What I mean by the release tarballs is to download https://files.pythonhosted.org/packages/4c/34/180b14dc521ae6a393c98d74d59afc79a7edc19a18721b69571843bfc495/cyipopt-1.2.0.tar.gz and extract it instead of checkout out a commit in the git repo.